### PR TITLE
added blog post linking to appearance on the enlightning podcast

### LINKED ISSUE #339: Show the video of enlightening podcast for Notary Project on the website
Notary Project maintainers just finished the livestream of the Enlightening podcast for Notary Project. It is worth showing this podcast video on our website.

See the podcast: https://tanzu.vmware.com/developer/tv/enlightning/56/

/cc @FeynmanZhou

### PR DESCRIPTION
**Pull request by zr-msft**
fixes #339

### CODE REVIEW

**Comment by zr-msft** [comment]
@FeynmanZhou @yizha1 @toddysm can you all please review?

**Review by yizha1 — APPROVED** [review]
LGTM

**Comment by zr-msft** [comment]
@toddysm  can you review please?

File: content/en/blog/2023/notary-enlightning-podcast.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: Notary Project featured on the Enlightning Podcast
+author:  "Notary Project maintainers"
+date: 2023-10-04
+draft: false
+---
+
+Notary Project maintainers Toddy Mladenov and Milind Gokarn talk with host Whitney Lee about the Notary Project on the [Enlightning Podcast](https://tanzu.vmware.com/content/videos/enlightning-ensuring-software-authenticity-introduction-to-notary-project). This wide-ranging discussion covers everything from the project history, to concepts, and to the future of the project.
```
Comment (FeynmanZhou) [inline content/en/blog/2023/notary-enlightning-podcast.md:8]: @zr-msft 
How about embedding the Youtube video player into this page? It brings a convenient experience to watch the video at notaryproject.dev. You can try embed the video by using the following code:

```
<iframe width="560" height="315" src="https://www.youtube.com/embed/FwpHZfNE9LQ?si=pzxe-qRM2z3oetwV" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```

Comment (zr-msft) [reply content/en/blog/2023/notary-enlightning-podcast.md]: good call! fixed

**Review by FeynmanZhou — APPROVED** [review]
LGTM

**Comment by zr-msft** [comment]
@yizha1 can you approve this PR so i can merge

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**